hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -205,7 +204,11 @@ private static void createBucket(String volumeName, String bucketName)
         .setVolumeName(volumeName)
         .setBucketName(bucketName)
         .build();
-    bucketManager.createBucket(bucketInfo);
+
```

Comment:
@bharatviswa504 can we make some test utility methods for this that hide the detail that table/cache are being updated directly?

Reply:
Done.